src/stores/filingHistoryListStore.ts

```diff
@@ -38,6 +38,17 @@ export const useFilingHistoryListStore = defineStore('filingHistoryList', {
       })
     },
 
+    /** Returns the total AGM extension duration */
+    getTotalAgmExtensionDuration (state: FilingHistoryListStateIF): number {
+      return state.filings.reduce((totalMonths, filing) => {
+        if (filing.name === FilingTypes.AGM_EXTENSION) {
+          // Add total months
+          totalMonths += filing.data?.agmExtension?.extensionDuration || 0
+        }
+        return totalMonths
+      }, 0)
+    },
```

Comment:
Very nice 👏

Reply:
Still nice!

Which line "cases year as number"? Do you need to do `return +totalMonths` somewhere?

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus